### DISCUSSION TITLE
Malware notification?

### DISCUSSION THREAD

**Discussion by gsparks3** [mtex-toolbox/mtex#2231] [Ask Anything] [answered]
I was recently informed by the IT department at work that `jcvoronoi.mexw64` in the MTEX 5.11.2 release was flagged as malware. I suspect this is a false positive, but I thought I would let you guys know so you can take a look. Sadly, I don't have access to much more information past that. Some info on what exactly set off the detector might have been helpful ...

Notably, they didn't care about the other versions of the same file (which I assume are for different operating systems, given the file extensions), only that one.

**Comment by ralfHielscher** [comment]
Thank you for letting me know. However, I have no idea what we can do about this. Maybe you can inform your IT department that this is open source software, that they can compile and check their self, and that their tools for detecting malware is not working correctly. I guess they are paying for it - so it makes more sense that their tool has to be fixed instead of MTEX.
Ralf.

**Reply by gsparks3** [reply]
Yeah, that's pretty much what I thought. 

To confirm: the relevant source code is here?
https://github.com/mtex-toolbox/mtex/tree/mtex-5.11.2/extern/jcvoronoi

**Reply by ralfHielscher** [reply, marked as answer]
yes exactly.
